tests/ignite/distributed/utils/__init__.py

```diff
@@ -228,6 +228,31 @@ def _test_distrib_barrier(device):
     assert tt.item() == true_res + 10.0
 
 
+def _test_distrib_new_group(device):
+    from ignite.distributed.comp_models import _SerialModel
+
+    bnd = idist.backend()
+    if idist.get_world_size() > 1:
+        ranks = [0, 1]
+        if idist.has_native_dist_support and bnd in ("nccl", "gloo", "mpi"):
+            from ignite.distributed.comp_models.native import _NativeDistModel
+
+            _model = _NativeDistModel()
+            assert _model.new_group(ranks).rank == dist.new_group(ranks=ranks).rank
```

Comment:
I think we should test it like:

```suggestion
            assert idist.new_group(ranks).rank == dist.new_group(ranks=ranks).rank
```